SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/ComponentTestUtils.h

```diff
@@ -70,6 +121,72 @@ inline bool CompareComponentData(const ComponentData& Lhs, const ComponentData&
 	return CompareSchemaObjects(Lhs.GetFields(), Rhs.GetFields());
 }
 
+inline bool CompareComponentChangeById(const ComponentChange& Lhs, const ComponentChange& Rhs)
+{
+	return Lhs.ComponentId < Rhs.ComponentId;
+}
+
+inline bool CompareComponentChanges(const ComponentChange& Lhs, const ComponentChange& Rhs)
+{
+	if (Lhs.ComponentId != Rhs.ComponentId)
+	{
+		return false;
+	}
+
+	if (Lhs.Type != Rhs.Type)
+	{
+		return false;
+	}
+
+	switch (Lhs.Type)
+	{
+	case ComponentChange::ADD:
+		if (!CompareSchemaComponentData(Lhs.Data, Rhs.Data))
```

Comment:
I think this is a case of if true return true else return false.
Can probably just return CompareSchemaComponentData